tests/tests_suite_test.go

```diff
@@ -364,6 +366,18 @@ func waitUntilDeployed() {
 	deploymentTimedOut = false
 }
 
+func waitUntilTemplateValidatorIsRunning() {
+	key := client.ObjectKey{Name: validator.VirtTemplateValidator, Namespace: strategy.GetNamespace()}
+	foundValidator := new(apps.Deployment)
+	Expect(apiClient.Get(ctx, key, foundValidator)).ToNot(HaveOccurred())
+
+	EventuallyWithOffset(1, func() bool {
+		Expect(apiClient.Get(ctx, key, foundValidator)).ToNot(HaveOccurred())
+
+		return foundValidator.Status.AvailableReplicas > 1
+	}, timeout, time.Second).Should(BeTrue())
```

Comment:
This function is called in `BeforeEach` for a lot of tests. In the worst case, this `EventuallyWithOffset()` will wait for 10 minutes for every test, which would take more than 9 hours.

Can you add a short-circuit logic similar to the one in `waitUntilDeployed()` function?